src/main/java/org/openstreetmap/atlas/geography/atlas/bloated/BloatedArea.java

```diff
@@ -36,25 +48,31 @@ public static BloatedArea fromArea(final Area area)
 
     public static BloatedArea shallowFromArea(final Area area)
     {
-        return new BloatedArea(area.getIdentifier()).withBounds(area.asPolygon().bounds());
+        return new BloatedArea(area.getIdentifier(), area.asPolygon());
     }
 
     BloatedArea(final long identifier)
     {
         this(identifier, null, null, null);
     }
 
+    BloatedArea(final long identifier, final Polygon polygon)
```

Comment:
Shouldn't that take just the bounds and not a full polygon?

Reply:
We could do that instead, and go back to your private `withOriginalBounds` method idea.